src/aap_eda/tasks/orchestrator.py

```diff
@@ -156,26 +156,27 @@ def dispatch(
     process_parent_id: int,
     request_type: Optional[ActivationRequest],
 ):
+    request_type_text = f" type {request_type}" if request_type else ""
```

Comment:
I would say we don't need this after https://github.com/ansible/eda-server/pull/863

Reply:
It's needed to avoid having `type None` appear in the message text.